tests/data/repos/conda_exercises_fleeqtk/fleeqtk_seq.xml

```diff
@@ -0,0 +1,102 @@
+<tool id="fleeqtk_seq" name="Convert to FASTA (fleeqtk)" version="0.1.0">
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <requirements>
+        <requirement type="package" version="1.3">fleeqtk</requirement>
+    </requirements>
+    <command><![CDATA[
+        fleeqtk seq
+              #if $settings.advanced == "advanced"
+              $settings.shift_quality
```

Comment:
Yeah - good plan. I think I want to update the seqtk tools at the same time. I created an issue #645.